uiowa/uiowa
# Off canvas tray bug fix

Resolves #7115.

# How to test
- checkout branch
- `ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli`
- Configure a layout builder block on an existing page or a add a new page and block and configure that
- See that the off-canvas tray for the block no longer takes up the entire screen. 
- Bonus: inspect the page and click the application tab. You should see a `ui_off_canvas_width` cookie with a value of 500
- You can also try adjusting the tray width, ~~but for me the change is only saved some of the time.~~ because of the patch the tray width is now saved correctly and actually updates as expected!
